doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -46,7 +46,7 @@ GeoServer can also produce vector tiles in three formats: GeoJSON, TopoJSON, and
      - This is a human readable JSON format.  Although many geo-spatial applications support GeoJSON datasets, few Vector Tile applications support tiles in this format.  Supported by Open Layers 3.
    * - `TopoJSON <https://github.com/mbostock/topojson/wiki>`_
      - ``application/json;type=topojson``
-     - This is a very complex, but somewhat human readable JSON format that is good for polygon coverages.  It is not a widely supported and very few Vector Tile applications support it.  Supported by Open Layers 3.
+     - This is a very complex, but somewhat human readable JSON format that is good for polygon coverages.  It is not a widely supported and very few Vector Tile applications support it.  Suported by Open Layers 3.
```

Comment:
Why this change?

Reply:
I have to be fair I am not too sure what happened there. I fixed it anyway

Reply:
It still appears to bad in the review version.  Can you check you have everything committed?